pkg/controller/vitesscluster/reconcile_topo.go

```diff
@@ -85,44 +85,53 @@ func (r *ReconcileVitessCluster) reconcileCellTopology(ctx context.Context, vt *
 		desiredCells[cell.Name] = cell
 	}
 
-	// We need to add an alias for all the cells in each region so that vtgate
-	// knows that it can route traffic between them.
-	// Currently, only one region is supported so we allow routing anywhere.
-	//
-	// We also need to create the aliases before we create the cells because we
-	// don't want any vtgates to start after the cells are created but before
-	// the alias exists.
-	desiredCellsAliases := buildCellsAliases(desiredCells)
-	currentCellsAliases, err := ts.GetCellsAliases(ctx, true)
+	if *vt.Spec.TopologyReconciliation.RegisterCellsAliases == true {
```

Comment:
The `== true` is already implicit in every conditional.

```suggestion
	if *vt.Spec.TopologyReconciliation.RegisterCellsAliases {
```

Reply:
Ahhh yeah, of course. This part of the code was a WIP - but thanks.  Tidied up now.